### ISSUE TITLE
Filter distort input

### ISSUE THREAD

**Issue by FreshTheFallen** [opentrack/opentrack#1124]
First time reporting an issue on GitHub, so please bear with me ;)

I'm using OpenTrack to transform my EdTracker input (as a Joystick) into TrackIR output, and have done so for half a year now. Everything worked fine until yesterday. Long story short, the output of Opentrack, esp. YAW and PITCH, are incredibly jumpy, e.g. I turn my EdTracker to 45° and hold it exactly there., the output goes to 45°, then jumps to 60°, back to 45°, 180°, 10°, etc.

The EdTracker is stationary at that moment, it is not moving. The raw data in OpenTrack confirms that, so does the system view, aswell as the EdTracker software. Only the OpenTrack output jumps randomly back and forth, but always returning back to 45° for a short moment. However, turning off the filter/not using ANY filter the output is perfectly fine and stays stationary.

In short, irregular output with any filters (tested with ACCELA, EWMA and Kalman) activated, stable when not using any filters.
OpenTrack v2.3.11
